modules/google-analytics.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/**
+ * Module Name: Google Analytics
+ * Module Description: MODULE DESCRIPTION COPY.
+ * First Introduced: 4.4
+ * Sort Order: 37
+ * Requires Connection: No
+ * Auto Activate: No
+ * Feature: Engagement
+ * Additional Search Queries: webmaster, google, analytics, console
+ */
+
+/**
+ * Removes the Google Analytics plugin settings page
+ */
+function removeSettingsPage() {
+	remove_submenu_page( 'options-general.php', 'wp-google-analytics' );
```

Comment:
If we're not going to use the existing settings code, could we remove completely instead of hiding it?